contents/handbook/growth/sales/how-we-work.md

```diff
@@ -84,3 +84,35 @@ Other metrics we care about:
 ## Travel to see customers
 
 You are likely to need to travel a lot more than the typical PostHog team member in order to meet customers. Please make sure that you follow our company [travel policy](/handbook/people/spending-money) and act in PostHog's best interests. We trust you to do the right thing here and won't pre-approve your travel plans, but we do keep track of what people are spending and the Ops team will follow up with you if it looks like you are wasting money here. We are not a giant company that pays for fancy flights, accommodation, and meals so please be sensible. 
+
+## Sales team guidelines
+
+### Key Principles
+- Understand our [ICP](https://posthog.com/handbook/who-we-are-building-for): Focus on understanding our users' needs and providing solutions that genuinely help them.
+- Build Credibility: Use PostHog to gain insights into user behavior and demonstrate how our products can solve their problems. Building trust and credibility through helpfulness and relevant information is essential.
+
+### Approach to Existing Accounts
+- Assess Account Activity: Monitor account activity to gauge interest. If an account shows high engagement (e.g., frequent usage, many active users), prioritize a more assertive approach.
+- Personalization: Tailor your outreach to the specific needs and usage patterns of the account. Highlight how our features address their needs.
+- Value-Driven Communication: Focus on providing value rather than just making a sale. Share insights and best practices relevant to their use case/industry.
+- Frequency: Regular but not intrusive. Aim for touchpoints every 2-3 weeks, adjusting based on account responses and activity.
+
+### Introducing Additional Products
+- Initial Onboarding: During the onboarding phase, mention additional products as part of the broader ecosystem but don't push for immediate adoption.
+- Usage Milestones: When an account reaches certain usage milestones or expresses satisfaction with the current product, introduce complementary products that can enhance their experience. Some of the milestones are:
```

Comment:
'introduce complementary products that can enhance their experience'

Do we have good examples here to make this less generic? e.g. if they are using X, then Y is more likely to make sense? I assume certain products pair more obviously together than others.